storage/resolver.go

```diff
@@ -52,23 +41,8 @@ func (resolver storageResolver) pipeRequestProxy(w http.ResponseWriter, r *http.
 	return
 }
 
-func addRequestHeadersToRequest(orig, forward *http.Request) {
-	for header, values := range orig.Header {
-		for _, value := range values {
-			forward.Header.Add(header, value)
-		}
-	}
-}
-
 func addRequestHeadersToResponse(w http.ResponseWriter, resp *http.Response) {
 	for header, values := range resp.Header {
-		if len(w.Header().Values(header)) > 0 {
-			// do not overwrite
-			continue
-		}
-		if _, ok := acceptedHeaders[header]; !ok {
-			continue
-		}
```

Comment:
I remember that I had to add the first check to avoid being `Cache-control` header being modified. Package registry adds its own cache-control headers, and package-storage CDN has other values.

Currently from package registry (https://github.com/elastic/package-registry/blob/main/handler.go#L33-L429)
```

Would that mean that Cache-control would have duplicated values ? 
 $ curl -Is https://epr.elastic.co/epr/security_detection_engine/security_detection_engine-8.4.1.zip |grep -i cache
cache-control: max-age=600,public
```
From package storage 
```bash
 $ curl -Is https://package-storage.elastic.co/artifacts/packages/security_detection_engine-8.4.1.zip  | grep -i cache
cache-control: public,max-age=3600
```